corehq/apps/sso/templates/sso/config_errors.html

```diff
@@ -29,20 +29,29 @@ <h2>
                   support so we can find a permanent solution.
                 {% endblocktrans %}
               </p>
+            {% elif is_oidc and error.code == error.OP_ERROR_MESSAGE %}
+              <p class="lead">
+                {% blocktrans with request.idp.service_name as service %}
+                  There was an issue logging in with {{ service }}:
+                {% endblocktrans %}
+              </p>
+              <p>
+                {{ error.message }}
```

Comment:
the error message does not contain user input

Reply:
I'm not sure why I flagged this. Even if this could contain user input, it's going to be escaped by default, since it's not going through the `safe` filter.

Reply:
as they say... better to be safe than sorry! ^_^